Dockerfile

```diff
@@ -26,4 +26,4 @@ USER root
 RUN find /code -user 0 -print0 | xargs -0 chown $USERNAME:$USERNAME
 USER $USERNAME
 
-CMD [ "yarn", "run", "start" ]
+CMD [ "node", "server.js" ]
```

Comment:
Can a comment be added here to explain why this is as it is so nobody tries to change it back to using Yarn to run something? Thanks!